pkg/config/config.go

```diff
@@ -46,8 +46,8 @@ type MetricConfig struct {
 }
 
 type ProviderConfig struct {
-	Source     map[string]SourceConfig `yaml:"source"`
-	Containerd ContainerdConfig        `yaml:"containerd"`
+	Source  map[string]SourceConfig `yaml:"source"`
+	WorkDir string                  `yaml:"work_dir"`
```

Comment:
Should also remove the `ContainerdConfig` structure definition.

Reply:
fixed